test/integration/fixtures/blocks/core__form-input__deprecated-v1.serialized.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:form-input -->
+<label class="wp-block-form-input__label">
+	<span class="wp-block-form-input__label-content">Label</span>
+	<input class="wp-block-form-input__input" type="text" name="label" aria-required="false"/>
+</label>
+<!-- /wp:form-input -->
```

Comment:
I expected new markup to be generated here, but strangely the markup hasn't changed. This may be related to the fact that the hook was not used in the save function.

Reply:
Your expectation was right, this would appear to indicate that the deprecation migration isn't running. At least for the fixture tests.